src/Console/Command/ResultsCommand.php

```diff
@@ -0,0 +1,333 @@
+                    if (!$hasParentTestcaseFataled && $test['status'] == AbstractPublisher::TEST_STATUS_STARTED) {
+                        $isInProgress = true;
+                    }
+
+                    $table->addRow([
+                        new TableCell(' - ' . $test['name']),
+                        $hasParentTestcaseFataled ? '' : $test['status'],
+                        $this->formatTestResult($test['result'], $test['status'], $tc['result']),
+                        $this->formatDate($test['start']),
+                        $hasParentTestcaseFataled ? '-' : $this->formatDate($test['end']),
+                        $this->formatDiff($test['start'], $test['end'], $isInProgress),
+                    ]);
+                }
+            }
+        }
+
+        $table->render();
+    }
+
+    /**
+     * @param \DateTimeInterface|null $start
+     * @param \DateTimeInterface|null $end
+     * @param bool $inProgress
+     * @return string
+     */
+    private function formatDiff($start, $end, $inProgress = false)
+    {
+        $output = '';
+
+        if ($inProgress && !$end) { // Is is in progress and end time is not available, use current time
+            $end = new \DateTimeImmutable();
+        }
+
+        if ($start instanceof \DateTimeInterface && $end instanceof \DateTimeInterface) {
+            $output = $end->getTimestamp() - $start->getTimestamp();
+
+            $output .= ' ' . ($inProgress ? '<options=blink>sec</>' : 'sec');
+        }
+
+        return $output;
+    }
+
+    /**
+     * @param \DateTimeInterface|null $date
+     * @return string
+     */
+    private function formatDate($date)
+    {
+        $output = '';
+        if ($date instanceof \DateTimeImmutable) {
+            $output = $date->format('Y-m-d G:i:s');
+        }
+
+        return $output;
+    }
+
+    /**
+     * @param string $status
+     * @param string $result
+     * @return string
+     */
+    private function formatTestcaseResult($status, $result)
+    {
+        $output = $result;
+
+        if ($status != ProcessWrapper::PROCESS_STATUS_DONE) {
+            return $output;
+        }
+
+        $colorMap = [
+            ProcessWrapper::PROCESS_RESULT_PASSED => '<fg=green>',
+            ProcessWrapper::PROCESS_RESULT_FAILED => '<fg=red>',
+            ProcessWrapper::PROCESS_RESULT_FATAL => '<fg=yellow>',
+        ];
+
+        if (isset($colorMap[$result])) {
+            $output = $colorMap[$result] . $output . '</>';
+        }
+
+        return $output;
+    }
+
+    /**
+     * @param string $result
+     * @param string $parentTestcaseResult
+     * @return string
+     */
+    private function formatTestResult($result, $status, $parentTestcaseResult)
+    {
+        $output = $result;
+
+        if ($parentTestcaseResult == ProcessWrapper::PROCESS_RESULT_FATAL
+            && $status == AbstractPublisher::TEST_STATUS_STARTED
+        ) {
+            return '<fg=yellow>fatal</>';
```

Comment:
I want to distinguish it from "usual" error, which uses red. I tried few other colors, and I doesn't look better :). Do you have any suggestion? In current revision I also [deduplicated](https://github.com/lmc-eu/steward/pull/65/files#diff-748943b5f72fe252742e36005a051907R30) the fatal error color definition to one place, so feel free to try some better color combination (maybe with background color?) if interested.

Also note same color could be used in the results.xml, which currently uses also yellow.